ConfigurationSystem/Client/CSCLI.py

```diff
@@ -9,6 +9,10 @@
 import sys
 import signal
 import types
+import atexit
+import os
+import readline
```

Comment:
The readline functionality is automatically picked up by the cmd module; I'm not aware of any other modules that are useable in that way.
readline seems to be included in all the externals bundles we inspected, so we assumed it would be ok to include it.